src/kiota/Handlers/KiotaGenerateCommandHandler.cs

```diff
@@ -27,6 +27,10 @@ public required Option<string> ClassOption
     {
         get; init;
     }
+    public required Option<string> ClassAccessModifierOption
```

Comment:
@andrueastman can you also provide guidance for the workspace commands so we have parity here please?

Reply:
That's right. 

@NKnusperer You'll also need to update the handlers here so that users with workspace can also use this option.
- https://github.com/microsoft/kiota/blob/main/src/kiota/Handlers/Client/AddHandler.cs
- https://github.com/microsoft/kiota/blob/main/src/kiota/Handlers/Client/EditHandler.cs

Once you read the option into the GenerationConfiguration you'll also need to update the `ApiClientConfiguration` so that this information is included in the workspace file.
- https://github.com/microsoft/kiota/blob/6675c2c48e07d988debf93da5f6367f712907ebc/src/Kiota.Builder/WorkspaceManagement/ApiClientConfiguration.cs#L65

As well as the comparer at 
https://github.com/microsoft/kiota/blob/main/src/Kiota.Builder/WorkspaceManagement/ApiClientConfigurationComparer.cs

Reply:
Implemented.